### PR TITLE
Add Terms And Conditions Page

### PR DESCRIPTION
**Pull request by roshni73**
## Addresses:
- https://github.com/IFRCGo/go-web-app/issues/1418

## Changes
- Add Terms and conditions page

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: app/src/views/AccountDetails/GenerateMontandonTokenModal/index.tsx [hunk 1]
```diff
@@ -75,9 +76,10 @@ function GenerateMontandonTokenModal(props: Props) {
                 {
                     termsLink: (
                         <Link
-                            external
-                            href="https://docs.google.com/spreadsheets/d/1kQZQmWk8W4tyE3fhi54Oc2Q-TiLJVUj0/"
-                            withLinkIcon
+                            to="termsAndConditions"
+                            actions={
+                                <ExternalLinkLineIcon />
```
Comment (samshara) [inline app/src/views/AccountDetails/GenerateMontandonTokenModal/index.tsx]: This is an internal link. So using `ExternalLinkLineIcon` could be confusing. Let's follow the pattern used in IFRC GO.

File: app/src/views/TermsAndConditions/i18n.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "namespace": "termsAndConditionsSS",
```
Comment (samshara) [inline app/src/views/TermsAndConditions/i18n.json]: Let's make sure to always verify our PRs ourselves first, as this seems to be a typo.

File: app/src/views/TermsAndConditions/i18n.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        "userRegistrationHeading":"USER REGISTRATION",
+        "userRegistrationDescription":"  Registration to this platform is open to staff, members and volunteers of the Red Cross Red Crescent Movement (National Societies, the IFRC and the ICRC) as well as to other responders, IFRC partners and authorised members of the public who agree to use the data and services for non-commercial purposes, such as research.",
+        "userRegistrationDescriptionToLink":"To request access and register, please use this link",
+        "dataProvenanceDescriptionAuthorizedUser":" Authorized Uses",
+        "userRegistrationDescriptionByRegistering":"By registering as a user to this platform, you will be able to:",
+        "userRegistrationDescriptionList1":"Have free access to the database"
```
Comment (samshara) [inline app/src/views/TermsAndConditions/i18n.json]: There are other items in the list as well. Did we miss them?

File: app/src/views/TermsAndConditions/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                            {strings.userRegistrationDescription}
+                        </p>
+                        <p>{strings.userRegistrationDescriptionToLink}</p>
+                        <p>
+                            {strings.dataProvenanceDescriptionAuthorizedUser}
+                        </p>
+                        <p>{strings.userRegistrationDescriptionByRegistering}</p>
+                        <ul>
+                            <li>{strings.userRegistrationDescriptionList1}</li>
+                            <li>Other uses that are not prohibited below</li>
```
Comment (samshara) [inline app/src/views/TermsAndConditions/index.tsx]: Is there any reason we aren't using strings here?

**Review by samshara — CHANGES_REQUESTED** [review]
This PR appears to be incomplete. Let's use Draft PRs for any unfinished work.

**Review by shreeyash07 — APPROVED** [review]

File: app/src/views/TermsAndConditions/CondtionsModal/i18n.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+    "namespace": "hereConditionsModal",
+    "strings": {
+        "hereConditionTitle":"TITLE",
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/CondtionsModal/i18n.json]: Please make it consistent.

File: app/src/views/TermsAndConditions/CondtionsModal/index.tsx [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+        filter: {},
+    });
+
+    const tableData: TableData[] = [
+        {
+            src_db_code: 'EMDAT',
+            src_db_lab: 'Emergency Events Database',
+            src_org_code: 'CRED',
+            src_org_lab: 'Centre for Research on the Epidemiology of Disasters (CRED) University of Louvain',
+            src_orgtype_code: 'orgtypeacad',
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/CondtionsModal/index.tsx]: Do we need translation here? Please verify it.

File: app/src/views/TermsAndConditions/i18n.json [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+        "protectionUsersPersonalDataHeading":"PROTECTION OF USER'S PERSONAL DATA",
+        "protectionUsersPersonalDataDescription":"The personal data required  for registration will be used by IFRC only for the purposes of ",
+        "protectionUsersPersonalDataDescriptionList1":"Validating your registration",
+        "protectionUsersPersonalDataDescriptionList2":"Providing access to the platform",
+        "protectionUsersPersonalDataDescriptionList3":"Providing information or any necessary communication regarding the platform(e.g.,a survey to assess how to improve the platform and better serve user's needs and provide a better experience for users)",
+        "protectionUsersPersonalDataDescriptionList4":"Providing aggregated metrics to partners so that they know how many people have viewed,accessed or downloaded their data via GO/Montandon",
+        "protectionUsersPersonalDataDescription2":"IFRC will not share your personal data. Any usage statistics reports shared with partners will not contain any personal data of users. If you no longer wish to be a user, you may request at any time that we delete your account by e-mail us at IM@ifrc.org.",
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/i18n.json]: Add mail to

File: app/src/views/TermsAndConditions/i18n.json [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+        "userRegistrationHeading":"USER REGISTRATION",
+        "userRegistrationDescription":"  Registration to this platform is open to staff, members and volunteers of the Red Cross Red Crescent Movement (National Societies, the IFRC and the ICRC) as well as to other responders, IFRC partners and authorised members of the public who agree to use the data and services for non-commercial purposes, such as research.",
+        "userRegistrationDescriptionToLink":"To request access and register, please use {termsLink} .",
+        "userRegistrationDescriptionLink":" this link",
+        "userRegistrationAuthorizedUser":" Authorized Uses ",
+        "userRegistrationDescriptionByRegistering":"By registering as a user to this platform, you will be able to:",
+        "userRegistrationLinkDescription":"information management -International Federation of Red Cross Crescent(IFRC).2024.Montandon - Global Crisis Data Bank:Revision 1.",
+        "userRegistrationByRegisteringLink":"http://montandon.wessteurope.cloudapp.azyre.com.8000/--doc--/. Accessed DAY MONTH YEAR.",
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/i18n.json]: Please verify it.

File: app/src/views/TermsAndConditions/i18n.json [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+        "protectionUsersPersonalDataDescriptionList4":"Providing aggregated metrics to partners so that they know how many people have viewed,accessed or downloaded their data via GO/Montandon",
+        "protectionUsersPersonalDataDescription2":"IFRC will not share your personal data. Any usage statistics reports shared with partners will not contain any personal data of users. If you no longer wish to be a user, you may request at any time that we delete your account by e-mail us at IM@ifrc.org.",
+        "protectionUsersPersonalDataDescription3":"IFRC will process and protect your personal data in accordance with its {termsLink}",
+        "protectionUsersPersonalDataLink3":" Policy on the Protection of Personal Data.",
+        "protectionUsersPersonalDataDescription4":"By agreeing to these Terms, you consent to the processing of your personal data by IFRC.",
+        "protectionUsersPersonalDataDescription5":"If you have any questions about how your personal data will be used, you may contact IM@ifrc.org.",
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/i18n.json]: Add mail to

File: app/src/views/TermsAndConditions/i18n.json [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+        "privilegesAndImmunitiesHeading":"PRIVILEGES AND IMMUNITIES",
+        "privilegesAndImmunitiesDescription":"Nothing in these Terms of Use or IFRC’s management of the database shall be deemed a limitation or waiver of any of the privileges and immunities of the Federation",
+        "finalProvisionsHeading":"FINAL PROVISIONS",
+        "finalProvisionsDescription1":"The IFRC reserves the right to amend or change these Terms at any time, at its sole discretion. You should consult them regularly to keep yourself informed of any changes.",
+        "finalProvisionsDescription2":"Any violation of these Terms may result in a temporary ban from the platform, and, where necessary, additional actions may be taken to address violations of these Terms.",
+        "finalProvisionsDescription3":"For more information on the database, its uses, data sources, limitations and peculiarities for data analaysis ot other related matters, please contact IFRC at{termsLink}",
+        "finalProvisionsDescription3Link":"IM@ifrc.org.",
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/i18n.json]: Add mail to

**Review by barshathakuri — CHANGES_REQUESTED** [review]

File: app/src/views/TermsAndConditions/CondtionsModal/i18n.json [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+        "srcDbCodeDescriptionTwenty": "DFO",
+        "srcDbLabDescriptionTwenty": "Dartmouth Flood Observatory (DFO)",
+        "srcOrgCodeDescriptionTwenty": "UniColumbia",
+        "srcOrgLabDescriptionTwenty": "University of Columbia",
+        "srcOrgtypeCodeDescriptionTwenty": "orgtypeacad"
+
+
+
+
+    }
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/CondtionsModal/i18n.json]: Remove this extra lines

File: app/src/views/TermsAndConditions/CondtionsModal/index.tsx [hunk 12]
```diff
@@ -0,0 +1,265 @@
...
+import useFilterState from '#hooks/useFilterState';
+
+import i18n from './i18n.json';
+
+interface Props {
+    onCancel: () => void;
+}
+
+interface TableData {
+    id:number;
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/CondtionsModal/index.tsx]:
```suggestion
    id: number;
```
make it consistent

File: app/src/views/TermsAndConditions/CondtionsModal/index.tsx [hunk 13]
```diff
@@ -0,0 +1,265 @@
...
+        sortState,
+    } = useFilterState<{
+        src_db_code?: string,
+        src_db_lab?: string,
+        src_org_code?: string
+        src_org_lab?: string
+        src_orgtype_code?: string
+    }>({
+        filter: {},
+    });
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/CondtionsModal/index.tsx]:
```suggestion
    src_org_lab: string;
    src_orgtype_code: string;
}

function ConditionalModal(props: Props) {
    const strings = useTranslation(i18n);
    const {
        onCancel,
    } = props;
    
    const {
        filtered,
        sortState,
    } = useFilterState<{
        src_db_code?: string,
        src_db_lab?: string,
        src_org_code?: string
        src_org_lab?: string
        src_orgtype_code?: string
    }>({
        filter: {},
    });
```
make it consistent

File: app/src/views/TermsAndConditions/i18n.json [hunk 14]
```diff
@@ -0,0 +1,78 @@
+{
+    "namespace": "termsAndConditions",
+    "strings": {
+        "termsAndConditionTitle": "Terms and Conditions",
+        "termsAndConditionsDescription1": "The IFRC GO is a Red Cross Crescent platform that amis to connect information on emergency needs with the the right response. This goal is achieved by making sure that the information – or data – we use is both accurate and accessible. Better data enables the Movement to respond more effectively to the needs of disaster-affected people.",
+        "termsAndConditionsDescription2": " At present  some data on the GO platform is visible to everyone. Access to certain National Society data requires registration and is restricted to the IFRC, ICRC, and the National Societies and is managed by IFRC.",
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/i18n.json]:
```suggestion
        "termsAndConditionsDescription2": "At present  some data on the GO platform is visible to everyone. Access to certain National Society data requires registration and is restricted to the IFRC, ICRC, and the National Societies and is managed by IFRC.",
```

File: app/src/views/TermsAndConditions/i18n.json [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+        "userRegistrationHeading": "USER REGISTRATION",
+        "userRegistrationDescription": "Registration to this platform is open to staff, members and volunteers of the Red Cross Red Crescent Movement (National Societies, the IFRC and the ICRC) as well as to other responders, IFRC partners and authorised members of the public who agree to use the data and services for non-commercial purposes, such as research.",
+        "userRegistrationDescriptionToLink": "To request access and register, please use {termsLink}.",
+        "userRegistrationDescriptionLink": "this link",
+        "userRegistrationAuthorizedUser": "Authorized Uses ",
+        "userRegistrationDescriptionByRegistering": "By registering as a user to this platform, you will be able to:",
+        "userRegistrationDescriptionList": "Information management -International Federation of Red Cross Crescent(IFRC).2024.Montandon - Global Crisis Data Bank:Revision 1.",
+        "userRegistrationByLink": "{termsLink}. Accessed DAY MONTH YEAR.",
+        "userMontandonLink": "http://montandon.westeurope.cloudapp.azure.com:8000/__docs__/",
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/i18n.json]: Why link in this file.

File: app/src/views/TermsAndConditions/i18n.json [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+        "unAuthorisedUsesDescription3": "Any reproduction or representation, whether in full or in part,for unauthorised purposes in prohibited.Failure to comply with this  provision will constitute a breach of these terms and may incur the infringer's liability.",
+        "complianceDateOwnerHeading": "COMPLIANCE WITH ANY APPLICABLE CONDITIONS FROM DATA OWNER",
+        "complianceDateOwnerDescription1": "When reusing any data  available in the Montandon database, it is user's responsibility to comply with any applicable terms of use or other conditions established by the data owner",
+        "complianceDateOwnerDescription2": "To facilitate when the data owner is not IFRC,IFRC makes the applicable terms of use from data source available {termsLink} and will, to the extent possible, keep it updated.",
+        "complianceDateOwnerDescription2Link": "here",
+        "complianceDateOwnerDescription3": "If unsure of rge applicable terms, you may contact IFRC at {termsLink} and IFRC can help liaise with the data owner.",
+        "complianceDateOwnerDescription3Link":"(IM@ifrc.org)",
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/i18n.json]: Why link in this file.

File: app/src/views/TermsAndConditions/i18n.json [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+        "protectionUsersPersonalDataDescription5": "If you have any questions about how your personal data will be used, you may contact {termsLink}.",
+        "privilegesAndImmunitiesHeading": "PRIVILEGES AND IMMUNITIES",
+        "privilegesAndImmunitiesDescription": "Nothing in these Terms of Use or IFRC’s management of the database shall be deemed a limitation or waiver of any of the privileges and immunities of the Federation",
+        "finalProvisionsHeading": "FINAL PROVISIONS",
+        "finalProvisionsDescription1": "The IFRC reserves the right to amend or change these Terms at any time, at its sole discretion. You should consult them regularly to keep yourself informed of any changes.",
+        "finalProvisionsDescription2": "Any violation of these Terms may result in a temporary ban from the platform, and, where necessary, additional actions may be taken to address violations of these Terms.",
+        "finalProvisionsDescription3":"For more information on the database, its uses, data sources, limitations and peculiarities for data analaysis ot other related matters, please contact IFRC at{termsLink}",
+        "finalProvisionsDescription3Link": "IM@ifrc.org.",
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/i18n.json]: Why link in this file.

File: app/src/views/TermsAndConditions/index.tsx [hunk 18]
```diff
@@ -0,0 +1,416 @@
...
+            'unauthorised-users': unauthorisedUsersRef,
+            'compliance-of-user-owner': complianceOfUserOwnerRef,
+            'liability-of-ifrc': liabilityOfIFRCRef,
+            'protection-personal-data': protectionPersonalDataRef,
+            'privileges-and-immunities': privilegesAndImmunitiesRef,
+            'final-provisions': finalProvisionsRef,
+        };
+
+        tabRefs[newTab]?.current?.scrollIntoView({ behavior: 'smooth' });
+    };
```
Comment (barshathakuri) [inline app/src/views/TermsAndConditions/index.tsx]:
```suggestion
        };
        tabRefs[newTab]?.current?.scrollIntoView({ behavior: 'smooth' });
    };
```

**Review by barshathakuri — DISMISSED** [review]

File: app/src/views/TermsAndConditions/CondtionsModal/i18n.json
Comment (tnagorra) [inline app/src/views/TermsAndConditions/CondtionsModal/i18n.json:1]: The filename has a typo.

File: app/src/views/TermsAndConditions/CondtionsModal/i18n.json [hunk 19]
```diff
@@ -0,0 +1,115 @@
+{
+    "namespace": "hereConditionsModal",
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/CondtionsModal/i18n.json]: Any reason the namespace is called "hereConditionsModal"?

File: app/src/views/TermsAndConditions/CondtionsModal/index.tsx [hunk 20]
```diff
@@ -0,0 +1,265 @@
...
+
+    const tableData: TableData[] = [
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/CondtionsModal/index.tsx]: Let's memoize this.

File: app/src/views/TermsAndConditions/CondtionsModal/index.tsx [hunk 21]
```diff
@@ -0,0 +1,265 @@
...
+            src_db_code: strings.srcDbCodeDescriptionTwenty,
+            src_db_lab: strings.srcDbLabDescriptionTwenty,
+            src_org_code: strings.srcOrgCodeDescriptionTwenty,
+            src_org_lab: strings.srcOrgLabDescriptionTwenty,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionTwenty,
+            id: 20,
+        },
+    ];
+
+    const tableKeySelector = (option:TableData) => option.id;
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/CondtionsModal/index.tsx]: Let's move this outside of the component. This is not dependent on any of the values in the component.

File: app/src/views/TermsAndConditions/index.tsx [hunk 22]
```diff
@@ -0,0 +1,416 @@
...
+export function Component() {
+    const strings = useTranslation(i18n);
+    const [showHereConditionModal, {
+        setTrue: setShowHereModalTrue,
+        setFalse: setShowHereModalFalse,
+    }] = useBooleanState(false);
+    type TitlesOptionKey = 'data-provenance' | 'user-registration' | 'unauthorised-users'| 'compliance-of-user-owner' | 'liability-of-ifrc' | 'protection-personal-data' | 'privileges-and-immunities' | 'final-provisions';
+    const [activeTitlesOption, setActiveTitleOption] = useState<TitlesOptionKey>('data-provenance');
+
+    const [accepted] = useBooleanState(false);
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/index.tsx]: Any reason `accepted` has been used as a constant?

File: app/src/views/TermsAndConditions/index.tsx [hunk 23]
```diff
@@ -0,0 +1,416 @@
...
+                        heading={strings.complianceDateOwnerHeading}
+                        withHeaderBorder
+                        withInternalPadding
+                        containerRef={complianceOfUserOwnerRef}
+                        footerContentClassName={styles.headerDescription}
+                        footerContent={(
+                            <>
+                                <p>{strings.complianceDateOwnerDescription1}</p>
+                                <p>
+                                    {!accepted && resolveToComponent(
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/index.tsx]: We should move the placement of the conditional checks. We most probably don't want to render empty <p> tags

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/src/views/TermsAndConditions/ConditionsModal/index.tsx [hunk 24]
```diff
@@ -0,0 +1,366 @@
...
+        {
+            src_db_code: strings.srcDbCodeDescriptionTwenty,
+            src_db_lab: strings.srcDbLabDescriptionTwenty,
+            src_org_code: strings.srcOrgCodeDescriptionTwenty,
+            src_org_lab: strings.srcOrgLabDescriptionTwenty,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionTwenty,
+            id: 20,
+        },
+    ], [
+        strings.srcDbCodeDescriptionEight,
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/ConditionsModal/index.tsx]: @frozenhelium What do you think of individually adding strings?

File: app/src/views/TermsAndConditions/ConditionsModal/index.tsx [hunk 25]
```diff
@@ -0,0 +1,366 @@
...
+        <Modal
+            heading={strings.hereConditionTitle}
+            onClose={onCancel}
+            size="xl"
+        >
+            <SortContext.Provider value={sortState}>
+                <Table
+                    filtered={filtered}
+                    columns={columns}
+                    keySelector={(item) => item.id}
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/ConditionsModal/index.tsx]: Let's not inline functions on render

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/src/App/routes/index.tsx [hunk 26]
```diff
@@ -612,6 +612,18 @@ const accountDetails = customWrapRoute({
...
+const termsAndConditions = customWrapRoute({
+    parent: rootLayout,
+    path: 'terms-and-conditions',
+    component: {
+        render: () => import('#views/TermsAndConditions'),
+        props: {},
+    },
+    context: {
+        title: 'Terms And Conditions',
+        visibility: 'is-authenticated',
```
Comment (tnagorra) [inline app/src/App/routes/index.tsx]: Is this page only accessible to logged-in users?

File: app/src/views/TermsAndConditions/ConditionsModal/index.tsx [hunk 27]
```diff
@@ -0,0 +1,266 @@
...
+        sortState,
+    } = useFilterState<{
+        srcDbCode?: string,
+        srcDbLab?: string,
+        srcOrgCode?: string
+        srcOrgLab?: string
+        srcOrgtypeCode?: string
+    }>({
+        filter: {},
+    });
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/ConditionsModal/index.tsx]: Why do we need useFilterState without using the filters?

File: app/src/views/TermsAndConditions/index.tsx [hunk 28]
```diff
@@ -0,0 +1,411 @@
...
+    const dataProvenanceRef = useRef<HTMLDivElement>(null);
+    const userRegistrationRef = useRef<HTMLDivElement>(null);
+    const unauthorisedUsersRef = useRef<HTMLDivElement>(null);
+    const complianceOfUserOwnerRef = useRef<HTMLDivElement>(null);
+    const liabilityOfIFRCRef = useRef<HTMLDivElement>(null);
+    const protectionPersonalDataRef = useRef<HTMLDivElement>(null);
+    const privilegesAndImmunitiesRef = useRef<HTMLDivElement>(null);
+    const finalProvisionsRef = useRef<HTMLDivElement>(null);
+
+    const handleTabChange = (newTab: TitlesOptionKey) => {
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/index.tsx:47]: Let's use useCallback

File: app/src/views/TermsAndConditions/index.tsx [hunk 29]
```diff
@@ -0,0 +1,411 @@
...
+                        footerContent={(
+                            <>
+                                <div>{strings.userRegistrationDescription}</div>
+                                <div>
+                                    {resolveToComponent(
+                                        strings.userRegistrationDescriptionToLink,
+                                        {
+                                            termsLink: (
+                                                <Link
+                                                    href="https://go.ifrc.org/register"
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/index.tsx]: This should not be an external link

File: app/src/views/TermsAndConditions/index.tsx [hunk 30]
```diff
@@ -0,0 +1,411 @@
...
+                                            termsLink: (
+                                                <Link
+                                                    href="mailto:IM@ifrc.org"
+                                                    external
+                                                    withUnderline
+
+                                                >
+                                                    IM@ifrc.org
+                                                </Link>
+                                            ),
```
Comment (tnagorra) [inline app/src/views/TermsAndConditions/index.tsx]: We have a lot of links to im@ifrc.org
Is this correct?

**Review by tnagorra — DISMISSED** [review]

File: app/src/views/TermsAndConditions/ConditionsModal/i18n.json [hunk 31]
```diff
@@ -0,0 +1,106 @@
+{
+    "namespace": "ConditionsModal",
```
Comment (samshara) [inline app/src/views/TermsAndConditions/ConditionsModal/i18n.json]:
```suggestion
    "namespace": "conditionsModal",
```

**Review by samshara — CHANGES_REQUESTED** [review]

File: app/src/App/routes/index.tsx (hunk 26 above)
Comment (AdityaKhatri) [reply app/src/App/routes/index.tsx]: no

File: app/src/views/TermsAndConditions/ConditionsModal/index.tsx (hunk 27 above)
Comment (AdityaKhatri) [reply app/src/views/TermsAndConditions/ConditionsModal/index.tsx]: For the sort, arigato!

**Review by samshara — APPROVED** [review]

**Review by samshara — APPROVED** [review]

**Pull request merged**